sql-parser/src/main/java/io/crate/sql/tree/Assignment.java

```diff
@@ -25,39 +25,64 @@
 import com.google.common.base.Objects;
 import com.google.common.base.Preconditions;
 
+import java.util.Collections;
+import java.util.List;
+
 public class Assignment extends Node {
 
     private final Expression columnName;
-    private final Expression expression;
-
+    private final List<Expression> expressions;
+
+    /**
+     * Constructor for SET SESSION/LOCAL statements
+     * one or more expressions are allowed on the right side of the assignment
+     * DEFAULT             -> empty list of expressions
+     * VALUE               -> single item in expressions list
+     *                        value can be either string literal, numeric literal, or ident
+     * VALUE, VALUE, ...   -> two or more items in expressions list
+     */
+    public Assignment(Expression columnName, List<Expression> expressions) {
+        Preconditions.checkNotNull(columnName, "columnname is null");
+        Preconditions.checkNotNull(expressions, "expression is null");
+        this.columnName = columnName;
+        this.expressions = expressions;
+    }
 
+    /**
+     * Constructor for SET GLOBAL statements
+     * only single expression is allowed on right side of assignment
+     */
     public Assignment(Expression columnName, Expression expression) {
         Preconditions.checkNotNull(columnName, "columnname is null");
         Preconditions.checkNotNull(expression, "expression is null");
         this.columnName = columnName;
-        this.expression = expression;
+        this.expressions = Collections.singletonList(expression);
     }
 
-
     public Expression columnName() {
         return columnName;
     }
 
     public Expression expression() {
-        return expression;
+        return expressions.isEmpty() ? null : expressions.get(0);
+    }
+
+    public List<Expression> expressions() {
+        return expressions;
     }
 
+
     @Override
     public int hashCode() {
-        return Objects.hashCode(columnName, expression);
+        return Objects.hashCode(columnName, expressions);
     }
 
 
     @Override
     public String toString() {
         return MoreObjects.toStringHelper(this)
             .add("column", columnName)
-            .add("expression", expression)
+            .add("expression", expression())
```

Comment:
print out expressions not expression, since this is not a field
i would also consider removing the expression method anyways